05 - coder-files/src/components/List.jsx

```diff
@@ -0,0 +1,11 @@
+export default function List({ array }) {
+  return (
+    <ul className="text-center">
+      {array.map((item, index) => (
+        <li key={index} className="py-3 border px-10 hover:bg-neutral-100">
+          {item}
+        </li>
+      ))}
+    </ul>
+  );
+}
```

Comment:
hey this is optional, but it can be good to name the array a more meaningful name such us `animalsList` for example